Musigwa/SendIT-API
# [WIP] #162082477 It updates the UI template to meet the feedback recommandations

## What does this PR do (?)

It updates the UI template to meet the feedback recommendations

## Description of Task to be completed (?)

The following should be working:

* updated descriptions for my stories- for both challenge 1 and challenge 2
* Used stock photos from unsplash.com
* 404 not found page (I created the page but I was unable to parse it properly as I told you in person)
* Re-integrated coveralls
* Obtained CI badges (e.g. from Code Climate and Coveralls) and add to ReadMe.

## How should this be manually tested (?)

After cloning this repo and switch to this branch,

* Using any RESTful Client tool (Usually the postman), test every endpoint, make sure to set your Headers as the following
  
Key: Content-Type value: application/json

* Navigate through the pages to see how they look like.
  
## Any background context you want to provide (?)

I'm using ES6 features in this branch, please be aware of the transpiler (.babelrc file) configurations.

## What are the relevant pivotal tracker stories (?)

1. 163082477
